src/main/java/org/terasology/Lakes/LakeProvider.java

```diff
@@ -68,35 +82,35 @@ public void process(GeneratingRegion region) {
         region.setRegionFacet(LakeFacet.class, lakes);
     }
 
-    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet){
+    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet) {
 
-        Rect2i BB = Rect2i.createFromMinAndMax(pos.x()-3,pos.z()-3,pos.x()+3,pos.z()+3);
+        Rect2i BB = Rect2i.createFromMinAndMax(pos.x() - 3, pos.z() - 3, pos.x() + 3, pos.z() + 3);
 
-        if(facet.getWorldRegion().contains(BB)){
-            float xDiff = Math.abs(facet.getWorld(pos.x()+3,pos.z())-facet.getWorld(pos.x()-3,pos.z()));
-            float yDiff = Math.abs(facet.getWorld(pos.x(),pos.z()+3)-facet.getWorld(pos.x(),pos.z()-3));
-            float xyDiff = Math.abs(facet.getWorld(pos.x()+3,pos.z()+3)-facet.getWorld(pos.x()-3,pos.z()-3));
-            if(xDiff > 2 || yDiff > 2 || xyDiff > 2){
+        if (facet.getWorldRegion().contains(BB)) {
+            float xDiff = Math.abs(facet.getWorld(pos.x() + 3, pos.z()) - facet.getWorld(pos.x() - 3, pos.z()));
+            float yDiff = Math.abs(facet.getWorld(pos.x(), pos.z() + 3) - facet.getWorld(pos.x(), pos.z() - 3));
+            float xyDiff = Math.abs(facet.getWorld(pos.x() + 3, pos.z() + 3) - facet.getWorld(pos.x() - 3,
+                    pos.z() - 3));
+            if (xDiff > 2 || yDiff > 2 || xyDiff > 2) {
                 return false;
-            }
-            else return true;
+            } else return true;
         }
 
         return false;
     }
 
-    protected boolean checkCorners(Rect2i BB, BaseFieldFacet2D facet){
+    protected boolean checkCorners(Rect2i BB, BaseFieldFacet2D facet) {
```

Comment:
While we're at it ^^

```suggestion
    protected boolean checkCorners(Rect2i boundingBox, BaseFieldFacet2D facet) {
```